frontend/public/clusters.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-undef */
+
+import { connect } from 'react-redux';
+import * as _ from 'lodash-es';
+import { Map as ImmutableMap } from 'immutable';
+
+export enum ClusterValues {
+  CLUSTER_ID = 'CLUSTER_ID'
+}
+
+const SET_CLUSTER_VALUE = 'setClusterValue';
+const DEFAULTS = _.mapValues(ClusterValues, undefined);
+export const setClusterValue = (name, value) => ({name, value, type: SET_CLUSTER_VALUE});
```

Comment:
I don't think this needs to be a map of values. Just have a single action that sets a cluster ID.